src/benchmarks/micro/libraries/System.Numerics.Vectors/Perf_VectorConvert.cs

```diff
@@ -0,0 +1,232 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Numerics.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD)]
+    public class Perf_VectorConvert
+    {
+        private const int Iterations = 1000;
+
+        // Vector.ConvertToXXX
+        [Benchmark]
+        public Vector<int> Convert_float_int() => Convert<float, int>(s_valuesFloat);
+
+        [Benchmark]
+        public Vector<uint> Convert_float_uint() => Convert<float, uint>(s_valuesFloat);
+
+        [Benchmark]
+        public Vector<long> Convert_double_long() => Convert<double, long>(s_valuesDouble);
+
+        [Benchmark]
+        public Vector<ulong> Convert_double_ulong() => Convert<double, ulong>(s_valuesDouble);
+
+        [Benchmark]
+        public Vector<float> Convert_int_float() => Convert<int, float>(s_valuesInt32);
+
+        [Benchmark]
+        public Vector<float> Convert_uint_float() => Convert<uint, float>(s_valuesUInt32);
+
+        [Benchmark]
+        public Vector<double> Convert_long_double() => Convert<long, double>(s_valuesInt64);
+
+        [Benchmark]
+        public Vector<double> Convert_ulong_double() => Convert<ulong, double>(s_valuesUInt64);
+
+        // Vector.Narrow
+        [Benchmark]
+        public Vector<float> Narrow_double() => Narrow<double, float>(s_valuesDouble);
```

Comment:
is it possible to narrow `double` only to `float`? If not, we should add the "to" type name to method name.

Reply:
Yep, that's the only double conversion supported by `Narrow`.  The `ConvertTo` methods have multiple overloads, so I qualified those names with both from and to types.  Could do that here as well for consistency if you think it helps.